frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestScheduledDialogs/ChangeRequestScheduledDialogue.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC, ReactElement } from 'react';
+import { Alert, styled, Typography } from '@mui/material';
+import { Dialogue } from '../../../common/Dialogue/Dialogue';
+
+export interface ChangeRequestScheduleDialogueProps {
+    title: string;
+    primaryButtonText: string;
+    open: boolean;
+    onConfirm: () => void;
+    onClose: () => void;
+    scheduledTime?: string;
+    message: string;
+    permissionButton?: ReactElement;
+    disabled?: boolean;
+    projectId?: string;
+    environment?: string;
+}
+
+const StyledAlert = styled(Alert)(({ theme }) => ({
+    marginBottom: theme.spacing(2),
+    backgroundColor: `${theme.palette.neutral.light}!important`,
+    color: `${theme.palette.text.primary}!important`,
+    borderColor: `${theme.palette.neutral.light}!important`,
+}));
+
+export const ChangeRequestScheduledDialogue: FC<
+    ChangeRequestScheduleDialogueProps
+> = ({
+    open,
+    onConfirm,
+    onClose,
+    title,
+    primaryButtonText,
+    message,
+    scheduledTime,
+}) => {
+    if (!scheduledTime) return null;
+
+    return (
+        <Dialogue
+            title={title}
+            primaryButtonText={primaryButtonText}
+            secondaryButtonText='Cancel'
+            open={open}
+            onClose={onClose}
+            onClick={() => onConfirm()}
+            fullWidth
+        >
+            <StyledAlert icon={false}>
+                There is a scheduled time to apply these changes set for{' '}
+                <strong>
+                    <br />
```

Comment:
Why are we using the `br` element? If we need to space this out, we should do that with styling, not html, I think.

Reply:
The reason I user `br` is that the designs look to me to require it be in a new line.  What would be a better way to do it?

Reply:
Yeah, this sounds like a styling thing to me. The way I've always learnt is that visual-only things should be done with CSS, not HTML. So for instance, if you need the `strong` element to be on its own line, you could use css to give it `display: block` instead. It's not a super important thing, but more of a ... best practices thing. At least as far as I'm aware.